pkg/controllers/provisioning/scheduling/nodeclaimtemplate.go

```diff
@@ -31,7 +31,9 @@ import (
 
 // MaxInstanceTypes is a constant that restricts the number of instance types to be sent for launch. Note that this
 // is intentionally changed to var just to help in testing the code.
-var MaxInstanceTypes = 100
+// Increasing this value needs caution as it also requires stress testing the cloud provider API to launch so many instances
```

Comment:
This is relevant only to AWS. I'd just leave the detail in the PR description and not include the comment

Reply:
Hmm, I will remove this but this could be true for any cloud provider right ?